src/bulk-load.ts

```diff
@@ -174,7 +223,25 @@ class BulkLoad extends EventEmitter {
     this.columnsByName[name] = column;
   }
 
-  addRow(...input: [ { [key: string]: any } ] | Array<any>) {
+  /**
+   * Adds a row to the bulk insert. This method accepts arguments in three different formats:
+   * <pre><code>bulkLoad.addRow( rowObj )
```

Comment:
This doc comment (and a bunch of the other ones) needs to be reformatted and converted from HTML to Markdown / typedoc.